cert_manager/README.md

```diff
@@ -0,0 +1,54 @@
+# Agent Check: cert_manager
+
+## Overview
+
+This check collects metrics from [cert-manager][1].
+
+![Cert-Manager Overview Dashboard][2]
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][3] for guidance on applying these instructions.
+
+### Installation
+
+The cert_manager check is included in the [Datadog Agent][3] package.
```

Comment:
Shouldn't "Datadog Agent" go to a different link? [3] goes to Kubernetes Integrations Autodiscovery.

Reply:
Thanks. I fixed it.